src/oc_erchef/apps/chef_index/README.md

```diff
@@ -6,7 +6,7 @@ Chef is a system integration framework written in erlang and ruby and designed t
 
 The Chef Wiki is the definitive source of user documentation.
 
-    http://wiki.chef.io/display/chef/Home
```

Comment:
The only line I think is important is the "put any changes that affect oc_reporting in chef_index".  I'll trim the rest.

Speaking of which...  I probably need to port some of this over don't I?